bindings/python/tests/conftest.py

```diff
@@ -15,9 +15,66 @@
 # specific language governing permissions and limitations
 # under the License.
 
+import os
+
 from dotenv import load_dotenv
 import pytest
+import opendal
 
 
 load_dotenv()
 pytest_plugins = ("pytest_asyncio",)
+
+
+def pytest_configure(config):
+    # register an additional marker
+    config.addinivalue_line(
+        "markers",
+        "need_capability(*capability): mark test to run only on named capability",
+    )
+
+
+@pytest.fixture()
+def service_name():
```

Comment:
What will happen if `OPENDAL_TEST` is empty or none?

Reply:
Value would be None, and the test would be failed

```text
ERROR tests/test_capability.py::test_capability - TypeError: argument 'scheme': 'NoneType' object cannot be converted to 'PyString'
ERROR tests/test_capability.py::test_capability_exception - TypeError: argument 'scheme': 'NoneType' object cannot be converted to 'PyString'
ERROR tests/test_read.py::test_sync_read - TypeError: argument 'scheme': 'NoneType' object cannot be converted to 'PyString'
ERROR tests/test_read.py::test_async_read - TypeError: argument 'scheme': 'NoneType' object cannot be converted to 'PyString'
ERROR tests/test_read.py::test_sync_read_stat - TypeError: argument 'scheme': 'NoneType' object cannot be converted to 'PyString'
ERROR tests/test_read.py::test_async_read_stat - TypeError: argument 'scheme': 'NoneType' object cannot be converted to 'PyString'
ERROR tests/test_read.py::test_sync_read_not_exists - TypeError: argument 'scheme': 'NoneType' object cannot be converted to 'PyString'
ERROR tests/test_read.py::test_async_read_not_exists - TypeError: argument 'scheme': 'NoneType' object cannot be converted to 'PyString'
ERROR tests/test_write.py::test_sync_write - TypeError: argument 'scheme': 'NoneType' object cannot be converted to 'PyString'
ERROR tests/test_write.py::test_async_write - TypeError: argument 'scheme': 'NoneType' object cannot be converted to 'PyString'
ERROR tests/test_write.py::test_sync_write_with_non_ascii_name - TypeError: argument 'scheme': 'NoneType' object cannot be converted to 'PyString'
ERROR tests/test_write.py::test_async_write_with_non_ascii_name - TypeError: argument 'scheme': 'NoneType' object cannot be converted to 'PyString'
ERROR tests/test_write.py::test_sync_create_dir - TypeError: argument 'scheme': 'NoneType' object cannot be converted to 'PyString'
ERROR tests/test_write.py::test_async_create_dir - TypeError: argument 'scheme': 'NoneType' object cannot be converted to 'PyString'
ERROR tests/test_write.py::test_sync_delete - TypeError: argument 'scheme': 'NoneType' object cannot be converted to 'PyString'
ERROR tests/test_write.py::test_async_delete - TypeError: argument 'scheme': 'NoneType' object cannot be converted to 'PyString'
```

Reply:
behavior tests should be skipped if OPENDAL_TEST is not set.

Reply:
Done

```text
tests/test_capability.py::test_capability SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                              [  6%]
tests/test_capability.py::test_capability_exception SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                    [ 12%]
tests/test_read.py::test_sync_read SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                                     [ 18%]
tests/test_read.py::test_async_read SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                                    [ 25%]
tests/test_read.py::test_sync_read_stat SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                                [ 31%]
tests/test_read.py::test_async_read_stat SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                               [ 37%]
tests/test_read.py::test_sync_read_not_exists SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                          [ 43%]
tests/test_read.py::test_async_read_not_exists SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                         [ 50%]
tests/test_write.py::test_sync_write SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                                   [ 56%]
tests/test_write.py::test_async_write SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                                  [ 62%]
tests/test_write.py::test_sync_write_with_non_ascii_name SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                               [ 68%]
tests/test_write.py::test_async_write_with_non_ascii_name SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                              [ 75%]
tests/test_write.py::test_sync_create_dir SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                              [ 81%]
tests/test_write.py::test_async_create_dir SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                             [ 87%]
tests/test_write.py::test_sync_delete SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                                  [ 93%]
tests/test_write.py::test_async_delete SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                                 [100%]
```

Reply:
Done

```text
tests/test_capability.py::test_capability SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                              [  6%]
tests/test_capability.py::test_capability_exception SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                    [ 12%]
tests/test_read.py::test_sync_read SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                                     [ 18%]
tests/test_read.py::test_async_read SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                                    [ 25%]
tests/test_read.py::test_sync_read_stat SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                                [ 31%]
tests/test_read.py::test_async_read_stat SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                               [ 37%]
tests/test_read.py::test_sync_read_not_exists SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                          [ 43%]
tests/test_read.py::test_async_read_not_exists SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                         [ 50%]
tests/test_write.py::test_sync_write SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                                   [ 56%]
tests/test_write.py::test_async_write SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                                  [ 62%]
tests/test_write.py::test_sync_write_with_non_ascii_name SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                               [ 68%]
tests/test_write.py::test_async_write_with_non_ascii_name SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                              [ 75%]
tests/test_write.py::test_sync_create_dir SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                              [ 81%]
tests/test_write.py::test_async_create_dir SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                             [ 87%]
tests/test_write.py::test_sync_delete SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                                  [ 93%]
tests/test_write.py::test_async_delete SKIPPED (OPENDAL_TEST not set)                                                                                                                                                                                                 [100%]
```